docs/cookbook/keys-descriptors/wallet-recovery.md

```diff
@@ -0,0 +1,18 @@
+# Wallet Recovery from Seeds
```

Comment:
This page is not needed IMO, as one needs descriptors to use and recover on BDK, and we address the mnemonic to descriptor conversions in the `From Seed Phrase to Descriptors` page.

Reply:
mmm ok, so I guess we could move to having the user copy paste descriptors instead of seed words... then this part of the full-wallet would just basically be the quickstart but with custom descriptors (using private keys). so we could replace the recovery section on the full-wallet page with a quickstart section (the purpose being to just check balance and generate an address)

Reply:
I didn't add this page, just renamed it and moved code to runnable crate